src/main/java/org/usrv/http/Server.java

```diff
@@ -106,6 +106,7 @@ private void handleRequest(Socket socket) {
                         response = new Response(200);
                         response.setHeader("Content-Type", file.getMimeType());
                         response.setBody(body);
+                        response.setHeader("Content-Length", String.valueOf(body.length()));
```

Comment:
I believe this will give the wrong value eg. when the String is in utf-8, since a char then takes up multiple bytes. A solution would be to:

- Change so we always read the file as bytes, not line by line
- Check the number of bytes and return that instead